heudiconv/convert.py

```diff
@@ -529,8 +529,9 @@ def save_converted_files(res, item_dicoms, bids_options, outtype, prefix, outnam
         #   echo times for all bids_files and see if they are all the same or not.
 
         # Check for varying echo times
+        from numpy import nan
```

Comment:
missed the PR, saw only commit in the issue, folded into #429 but adjusted commit to use math.nan not numpy.nan here